lib/core/presentation/pages/chat/chat_message/widgets/message_item/message_reaction_widget.dart

```diff
@@ -0,0 +1,228 @@
+import 'package:app/core/presentation/pages/chat/chat_message/widgets/chat_input/emoji_picker_widget.dart';
+import 'package:app/core/presentation/widgets/chat/matrix_avatar.dart';
+import 'package:app/core/service/matrix/matrix_service.dart';
+import 'package:app/core/utils/bottomsheet_utils.dart';
+import 'package:app/injection/register_module.dart';
+import 'package:app/theme/color.dart';
+import 'package:app/theme/sizing.dart';
+import 'package:app/theme/spacing.dart';
+import 'package:flutter/material.dart';
+import 'package:matrix/matrix.dart';
+import 'package:collection/collection.dart' show IterableExtension;
+
+class MessageReactions extends StatelessWidget {
+  final Event event;
```

Comment:
Add `widget` suffix

Reply:
@sieuhuflit may need your opinion on this ? 
We already had Bloc | Page | Utils as a suffix for bloc and page and other utils. Then the rest obviously widget, can consider if we still need Widget as a suffix ? 
Personally I prefer naming without Widget.